_data/success-stories.yml

```diff
@@ -25,6 +25,11 @@ cases:
     logo: cases-logos/waze.jpg
     text: Thanks to Spinnaker, developers can focus on developing business logic rather than becoming experts on each cloud platform. Teams can track the lifecycle of a deployment using several notification mechanisms including email, Slack, and SMS, allowing them to coordinate handoffs between developer and QA teams.
 
+  - name: Biden for President
+    link: https://spinnaker.io/guides/
```

Comment:
Wondering why this link is to the Spinnaker docs? In other Case Studies, the "Learn" link goes to a whitepaper or external site with an actual case study. Is it possible to link to wherever that quote appeared?  twitter, news site, ??